src/components/IntegrationTest/tabs/IntegrationTestOverviewTab.tsx

```diff
@@ -85,7 +85,7 @@ const IntegrationTestOverviewTab: React.FC<IntegrationTestOverviewTabProps> = ({
               <DescriptionListGroup>
                 <DescriptionListTerm>Image bundle</DescriptionListTerm>
                 <DescriptionListDescription>
-                  <ExternalLink href={integrationTest.spec.bundle}>
+                  <ExternalLink href={`https://${integrationTest.spec.bundle}`}>
```

Comment:
Looks like it's possible to create integration test with bundle url already having http, should we add a check in that case?